rfc/cdevents-translator.md

```diff
@@ -1,21 +1,38 @@
 ## CDEvents Translator
 
 ### Overview:
-This design describes the approaches to translate events from various source control systems into CDEvents and sending them to configured message-broker, by using a common `cdevents-translator`.
+This design describes the approaches to translate events from various Source Code Management (SCM) systems into CDEvents and sending them to configured message-broker, by using a common `cdevents-translator`.
 
-A library `cdevents-translator` will have an interface that can be implemented for various SCM systems like GitHub, Gerrit and Gitlab to translate events into CDEvents format.
+A library `cdevents-translator` will have an interface that can be implemented for various Source Code Management (SCM) systems like GitHub, Gerrit and Gitlab to translate events into CDEvents format.
 
 Different SCM systems or applications that are exposing events with endpoints/webhooks can be configured and mapped with the corresponding CDEvent.
 
 For Example configuring Gerrit webhooks and mapping with CDEvents can be found in [gerrit-cdevent](gerrit-cdevents.md) 
 
+### Motivation and Rationale
+In establishing the fundamental principles of CDEvents, a key objective is to ensure that events are sent directly from their source or as close to the source as feasible.</br> 
+Recognizing the challenge of achieving universal adaptation among all tools and systems to send CDEvents, there arises a need to bridge this gap.</br>
+To address this, the `cdevents-translator` will be implemented. This translator is designed to translate and send CDEvents to a message-broker, for various SCM systems.
+
+### Goals
+
+- The translators will be implemented for tools that support sending events over the HTTP protocol with webhook capabilities
+- Implement support for one-way translation only; that is, translating events from different SCM systems to CDEvents
+- The translators will be implemented to translate only [Source Code Control](https://cdevents.dev/docs/source-code-version-control/) type CDEvents
+- Create shared packages/interfaces that will be commonly used for translating and sending CDEvents. 
+- Finalize one of the three approaches and utilize the `cdevents-translator` library to implement the Gerrit translator initially.
+
+### Non-Goals
+- Implement translators for other SCM systems or any applications requiring translation to CDEvents
```

Comment:
I thought this proposal intends to potentially provide support for any SCM system, but no other system
```suggestion
- Implement translators for other applications (non-SCM systems) requiring translation to CDEvents
```